src/test/ui/parser/recover-where-clause-before-tuple-struct-body.stderr

```diff
@@ -0,0 +1,51 @@
+error: where clauses are not allowed before tuple struct bodies
+  --> $DIR/recover-where-clause-before-tuple-struct-body.rs:4:1
+   |
+LL | / where
+LL | |     (): Sized,
+   | |______________^
+LL |   (usize);
+   |   ------- the tuple struct body
+   |
+help: consider moving the where clause after the body
+   |
+LL - where
+LL + (usize) where (): Sized;
+   |
+
+error: where clauses are not allowed before tuple struct bodies
+  --> $DIR/recover-where-clause-before-tuple-struct-body.rs:10:1
+   |
+LL | / where
+LL | |     (): Sized,
+LL | |     String: Clone,
+   | |__________________^
+LL |   (pub usize, usize);
+   |   ------------------ the tuple struct body
+   |
+help: consider moving the where clause after the body
+   |
+LL - where
+LL + (pub usize, usize) where (): Sized, String: Clone;
```

Comment:
Personally, I am not a fan how this diff is rendered. In my eyes, it looks as if it's only the `where` keyword that needs to be replaced and as if the trait bounds are allowed to stay.
I would have expected it to like this (nested diff incoming):

```diff
- - where
- + (pub usize, usize) where (): Sized, String: Clone;
+ - where
+ -     (): Sized,
+ -     String: Clone,
+ + (pub usize, usize) where (): Sized, String: Clone;
```

I've double-checked that the span is correct by running `rustfix`, so it's only the rendering that's off.

Reply:
File a ticket for it. It is definitely in the `Emitter`.